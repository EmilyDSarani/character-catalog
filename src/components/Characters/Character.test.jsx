import { render } from '@testing-library/react'
import Character from './Character'

it('should render character', () => {
    const { container } = render(
    <Character character={{
        image: 'https://place-puppy.com/',
        name:'Sarani',
        species:'Humanoid',
        status:'possibly alive'}} />);
    expect(container).toMatchSnapshot();
});