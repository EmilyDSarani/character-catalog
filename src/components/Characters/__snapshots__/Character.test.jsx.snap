// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render character 1`] = `
<div>
  <figure>
    <img
      alt="Sarani"
      src="https://place-puppy.com/"
    />
    <figcaption>
      <h3>
        Sarani
      </h3>
      <p>
        Humanoid
      </p>
      <p>
        Status: 
        possibly alive
      </p>
    </figcaption>
  </figure>
</div>
`;
